server/routes/projectRoutes.js

```diff
@@ -9,7 +10,8 @@ const {
 const auth = require('../middleware/authMiddleware');
 
 router.get('/', getAllProjects); // Get all projects (public)
-router.get('/:userId/', getProjectsForUser); // Get all projects for a specific user
+router.get('/:projectId', getProject);
+//router.get('/:userId', getProjectsForUser); // Get all projects for a specific user
```

Comment:
Is there a reason you removed this route?

Reply:
Actually it was creating a conflict between getProject and getProjectForUser, both being get requests. Maybe we can change id to a query parameter?